deploy.go

```diff
@@ -274,9 +274,6 @@ func setRootFuncInfo(ff *funcfile, appName string) {
 
 func (p *deploycmd) updateRoute(c *cli.Context, appName string, ff *funcfile) error {
 	fmt.Printf("Updating route %s using image %s...\n", ff.Path, ff.ImageName())
-	if err := resetBasePath(p.Configuration); err != nil {
```

Comment:
not sure what this was? double checking it's ok?

Reply:
makes sense i think, doing this at client level now

Reply:
yeah, it's on the responsibility of a client, this method was quite confusing and its purpose what not clear, it was just appending the value of API_URL and `v1`, so fn_go does this every time we're hitting any of its API.